web/modules/custom/vote_utility/translations/ja.po

```diff
@@ -138,10 +138,10 @@ msgid "Register by mail deadline:"
 msgstr "郵送での登録締切："
 
 msgid "Must be postmarked by @date"
-msgstr "2022年5月28日 @date 消印有効"
+msgstr "年5月28日 @date 消印有効"
```

Comment:
The entire hardcoded date should be removed

Reply:
removed!